pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -7,6 +7,7 @@
 , brotli, cairo, pixman, xorg, clisp, biber, woff2, xxHash
```

Comment:
(commenting at random position so it can be marked resolved)

the binaries-test for upmendex now fails:
```
error: builder for '/nix/store/9f6nqbzlvlyq0ir1h078lylyj64bsc69-texlive-test-binaries.drv' failed with exit code 1;
       last 7 log lines:
       > failed 'upmendex --version' (exit code: 255)
       >   > warning: kpathsea: configuration file texmf.cnf not found in these directories: /nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/bin:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/bin/share/texmf-local/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/bin/share/texmf-dist/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/bin/share/texmf/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/bin/texmf-local/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/bin/texmf-dist/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/bin/texmf/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/share/texmf-local/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/share/texmf-dist/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/share/texmf/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/texmf-local/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/texmf-dist/web2c:/nix/store/h97bll48d5l126g9v575vsqck7ll7cca-texlive-core-big.bin-2022-uptex/texmf/web2c:/nix/texmf-local/web2c:/nix/store:/nix/store/share/texmf-local/web2c:/nix/store/share/texmf-dist/web2c:/nix/store/share/texmf/web2c:/nix/store/texmf-local/web2c:/nix/store/texmf-dist/web2c:/nix/store/texmf/web2c.
       >   > This is upmendex version 1.00 [ICU 73.2] (TeX Live 2022).
       >   > Scanning input file stdin....done (0 entries accepted, 0 rejected).
       >   > 0 entries accepted, 0 rejected.
       >   > Nothing written in output file.
       > tested 439 binaries: 17 ignored, 2 broken, 1 failed
       For full logs, run 'nix log /nix/store/9f6nqbzlvlyq0ir1h078lylyj64bsc69-texlive-test-binaries.drv'.
```
so it should probably be marked as „requires combined env“.

Reply:
I believe this is solved now, `upmendex --help` works without a scheme apparently. I couldn't run the test proper because thanks to this PR, I either recompile asymptote, or both cores, neither of which is that great on this machine.

Reply:
can confirm that it works now